tests/e2e/restart_test.go

```diff
@@ -84,7 +84,7 @@ func (s *RestartTestData) TestVolumeDetached(c *C) {
 	c.Assert(err, IsNil, Commentf(out))
 
 	// 4. Verify detached status. Volume should be detached (within the timeout)
-	status = verification.VerifyDetachedStatus(s.volumeName, s.config.DockerHosts[1], s.config.EsxHost)
+	status = verification.PollDetachedStatus(s.volumeName, s.config.DockerHosts[1], s.config.EsxHost)
```

Comment:
We've never seen it in any of the tests till date on CI at least. If anything thats a bug or the thread synch errors we have seen in the past. How can they give different results when the source is exactly the same. If anything those bugs have to be fixed. No user is going to do double verification when all they would need is docker volume inspect and it better be correct.